### PR TITLE
Update network dumps docs

### PR DESCRIPTION
**Pull request by jonbarrow**
Replaces the submission section to use the new slash command. Also tweaks some existing docs which, upon further inspection, seemed to need some changes/clarifications

### CODE REVIEW

File: docs/en_US/network-dumps.md [hunk 1]
```diff
@@ -18,14 +18,13 @@ In order to make this easier, we have developed a suite of internal tools to cap
...
+1. Perform as few actions as possible per session. Each session should be focussed on as few aspects of the game as possible to minimize "noise" (unrelated packets) which makes it difficult to decipher what is actually going on. For example, rather than uploading a session of playing random courses on Super Mario Maker it would be more helpful if the session was focussed on one part of the game, such as 100 Mario Challenge, starring a course, etc. That way it's clear exactly what the packets are doing in game.
+2. When submitting, please provide a detailed description of what was done in the session. For example "Loaded course world and played a random course, starred the course and played a recommended course". This will add further context as to what the packets are doing in the dumps.
+3. When submitting, please rename the dumps to something meaningful. Such as `played-and-starred-course-smm-jpn.pcap` rather than `did-something.pcap` or `12-20-23.pcap`. This does not apply to HokakuCafe BIN files, as they must remain named as they are. The file name of HokakuCafe BIN files gives us your NEX username (PID).
```
Comment (DaniElectra) [inline docs/en_US/network-dumps.md:27]: > The file name of HokakuCafe BIN files gives us your NEX username (PID).

Well tecnically we can get the PID from the `Login`/`LoginEx` requests. Having the game server ID is useful though

Comment (jonbarrow) [reply docs/en_US/network-dumps.md:27]: True. Do you suggest we change this?

Comment (DaniElectra) [reply docs/en_US/network-dumps.md:27]: Maybe something like "The file name of HokakuCafe BIN files gives us more information about the game servers."?

Comment (jonbarrow) [reply docs/en_US/network-dumps.md]: Updated :+1:

**Pull request merged**